common/rootfs/docker-entrypoint.sh

```diff
@@ -8,6 +8,20 @@ bashio::log.info "Preparing to start..."
 # - https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/387
 bashio::config.require 'data_path'
 
+export ZIGBEE2MQTT_DATA="$(bashio::config 'data_path')"
+
+# Migrate configuration to addon specific data path for HA backups, see https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/627
+if [[ "$ZIGBEE2MQTT_DATA" == "/addon_config/zigbee2mqtt" ]] && ! bashio::fs.file_exists "$ZIGBEE2MQTT_DATA/configuration.yaml" && bashio::fs.file_exists "/config/zigbee2mqtt/configuration.yaml"; then
```

Comment:
Looks like the folder is named `addon_configs` not `addon_config`?
(same for other refs in PR)

Reply:
The config folder is mounted to `/addon_config/` as specified in the [config.json](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/blob/421c04c8084b90dde64e124ffe2603d740615f6f/zigbee2mqtt/config.json#L38)